### PR TITLE
Add SSL redirect when using `redirect_from`

### PR DESCRIPTION
**Pull request by carlalexander**

### CODE REVIEW

**Comment by carlalexander** [comment]
As discussed, this adds SSL redirect when using `redirect_from`. I'm not sure this is the best way to handle it it, but it covers my basic use case.

File: templates/etc/nginx/sites-available/default.conf.j2 [hunk 1]
```diff
@@ -423,6 +423,14 @@ server {
 {%             for port in nginx_tpl_listen              %}
         listen {{ port|string }};
 {%             endfor                                    %}
+{%             if nginx_tpl_ssl | bool                   %}
+{%                 for port in nginx_tpl_listen_ssl      %}
+        listen {{ port|string }};
```
Comment (drybjed) [inline templates/etc/nginx/sites-available/default.conf.j2]: This line [according to the documentation](https://nginx.org/en/docs/http/configuring_https_servers.html#single_http_https_server) should be:
```
listen {{ port|string }} ssl;
```
Can you check if this works as expected?

Comment (carlalexander) [inline templates/etc/nginx/sites-available/default.conf.j2]: It worked when I tested, but let me fix it.

**Comment by carlalexander** [comment]
Fixed the issue!

**Pull request merged**